1.2/sd-client/pom.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ * Copyright 2014 Cisco Systems, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>com.cisco.oss.foundation.directory</groupId>
+    <artifactId>sd-common-pom</artifactId>
+    <version>1.0.5</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>sd-client</artifactId>
+  <name>Service Directory Client Application</name>
+  <version>1.2.1-0-SNAPSHOT</version>
```

Comment:
Suggest to use the exact same version as the sd-api. This way we know immediately what SD API version the client uses.